content/blogposts/2022/multi-version-upgrades.md

```diff
@@ -0,0 +1,108 @@
+---
+title: "Multi-version upgrades"
+description: "Placeholder."
+authors:
+  - name: Eric Fritz
+    url: https://eric-fritz.com
+publishDate: 2022-09-09T18:00+02:00
+tags: [blog]
+slug: multi-version-upgrades
+heroImage: placeholder
+socialImage: placeholder
+published: true
+---
+
+It's finally happened.
+
+The most obnoxious restriction (by a very wide margin) related to administration of a Sourcegraph instance no longer applies. Sourcegraph instances (v3.20 or newer) are now able to upgrade directly to a future release, without the requirement to hit every minor release along the way.
+
+A recent peek at our customer instance distribution showed that 70% of our self-hosted customers are on a "historic" release of Sourcegraph (older than two months). Larger customer instances also tend to approach upgrades a bit more cautiously, adding significant weight to the use of older releases. Note that [Sourcegraph v3.20](https://about.sourcegraph.com/blog/release/3.20) was released fairly _early_ in the most recent pandemic era, and we've spent the last two years of that era shipping improvements and building new features and workflows for an on-prem audience that was largely unable to benefit from the effort.
+
+[Multi-version upgrades](https://docs.sourcegraph.com/admin/updates#multi-version-upgrades), contrasting to the "standard upgrade" approach, use the [`migrator` tool](introducing-migrator-service) to perform the sequence of schema and in-place data migrations that _would happen_ if the user were to perform a chain of upgrades that hit every minor release on the way to their target. This tool makes upgrading directly to our [4.0 release](TODO:FILLMEIN) a technical feasibility for the vast majority of our customers.
```

Comment:
Ensure we update the placeholder link targett.

Reply:
https://about.sourcegraph.com/blog/release/4.0